File: people/thom/test/makeptd.f
CC=cc
FORT=f90 
MC=../mcsrc/mc
BASE_PATH=..
CFLAGS=-g 
FFLAGS=-free
CLIBS = -lf90
CINCLUDES = -L/usr/local/lib/f90 -I/usr/local/lib/f90 -L/usr/local/lib  

all:    ptd doerror ptde.dat

ptde.dat: ptdm.dat 
	doerror ptdm.dat ptdf.dat ptde.dat 5

doerror: error.c
	$(CC) $(CFLAGS) error.c -o doerror

ptdm.dat: ptd 
	ptd ptdm.dat ptdf.dat

ptd: ptdmain.o ptdf.o ptdm.o
	$(FORT) $(FFLAGS) $(CFLAGS) $(CINCLUDES) -o ptd ptdmain.o ptdf.o ptdm.o 

ptdmain.o: ptdmain.f
	$(FORT) $(FFLAGS) $(CFLAGS) $(CINCLUDES) -c ptdmain.f

ptdf.o: ptdf.f
	$(FORT) $(FFLAGS) $(CFLAGS) $(CINCLUDES) -c ptdf.f

ptdm.o: ptdm.f
	$(FORT) $(FFLAGS) $(CFLAGS) $(CINCLUDES) -c ptdm.f

ptdm.f: ptdm.m
	$(MC) ptdm.m -o ptdm.f

clean:
	rm -f ptdf.o ptdm.o ptdmain.o ptdm.dat ptdf.dat ptde.dat ptdm.f ptd

File: people/thom/test/bak/makeptd.c
CC=cc
MC=../mcorg/mc
BASE_PATH=..
CFLAGS=-g 

all:    ptd doerror ptde.dat

ptd: ptdmain.o ptdc.o ptdm.o
	$(CC) $(CFLAGS) -o ptd ptdmain.o ptdc.o ptdm.o 

ptde.dat: ptdm.dat ptdc.dat
	doerror ptdm.dat ptdc.dat ptde.dat 5

doerror: error.c
	$(CC) $(CFLAGS) error.c -o doerror

ptdm.dat ptdc.dat:
	ptd ptdm.dat ptdc.dat

ptdmain.o: ptdmain.c
	$(CC) $(CFLAGS) -c ptdmain.c

ptdc.o: ptdc.c
	$(CC) $(CFLAGS) -c ptdc.c

ptdm.o: ptdm.c
	$(CC) -c ptdm.c

ptdm.c: ptdm.m
	$(MC) ptdm.m -o ptdm.c

clean:
	rm -f ptdc.o ptdm.o ptdmain.o ptdm.dat ptdc.dat ptde.dat ptdm.c ptd

